app/views/guide_typography.html

```diff
@@ -133,9 +133,11 @@ <h3 class="heading-medium" id="typography-links">Links</h3>
 </pre>
 
   <h3 class="heading-medium" id="typography-lists">Lists</h3>
-  <p class="text">
-    List items start with a lowercase letter and have no full stop at the end.
-  </p>
+  
+  <ul class="list list-bullet text">
+    <li>list items start with a lowercase letter and have no full stop at the end</li> 
```

Comment:
Given the conversation in the other PR, can I suggest rather than duplicating we just defer to the style guide here, like we do in the markdown documentation?

e.g.

> See the [style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps) for guidance on how to punctuate lists.

We'd still update the examples.